fake/fake-extension.js

```diff
@@ -80,6 +80,31 @@ var seedMvcActions = (function() {
                         { tag: 'id', match: id, default: null, required: false }
                     ]
                 };
+            },
+            headers: function(host, contentType) {
```

Comment:
Style nit/question: Does this code get compiled via TypeScript (or Babel)? If so, we can use the new ES2015 shorthand syntax:

```
headers(host, contentType) {
...
```